### PR TITLE
Fixed Errors in README.md

### PR DESCRIPTION
**Pull request by tdslinden**
- Hedging
- Remove pronouns for clarity
- Removed excess use of that
- Removed contractions to make more formal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,16 +17,16 @@ A collection of code snippets built with [Script Lab](//github.com/OfficeDev/scr
 4. Install `yarn` as a global package `npm install yarn --global`.
 5. Be sure your CLI is in the root of the office-js-snippets repo and run `yarn install`. (It is similar to `npm install`.)
 6. Set up the original \OfficeDev\office-js-snippets as the upstream repo for your local repo by following the steps in [Configuring a remote for a fork](https://help.github.com/articles/configuring-a-remote-for-a-fork/).
-7. If you'll be using Visual Studio Code as your editor, install the [TSLint](https://marketplace.visualstudio.com/items?itemName=eg2.tslint) extension for Visual Studio Code.
```
Comment (Rick-Kirkham) [inline README.md]: Current Microsoft style is to use contractions. It makes the text seem less formal and friendlier.

File: README.md [hunk 2]
```diff
@@ -17,16 +17,16 @@ A collection of code snippets built with [Script Lab](//github.com/OfficeDev/scr
...
+7. If you will be using Visual Studio Code as your editor, install the [TSLint](https://marketplace.visualstudio.com/items?itemName=eg2.tslint) extension for Visual Studio Code.
 
 ### Adding a new sample
 
-> For the git tasks in this procedure, the instructions assume that you are using a CLI. You are welcome to use a GUI git client. Consult the client's help to learn how to carry out the same tasks.
```
Comment (Rick-Kirkham) [inline README.md]: Including the "that" at the beginning of a relative clause makes it easier for readers for whom English is a second language.

File: README.md [hunk 3]
```diff
@@ -35,12 +35,12 @@ api_set:
...
 7. Make sure the master branch of your fork is in sync with the master branch of the upstream \OfficeDev\office-js-snippets repo by following the steps in [Syncing a fork](https://help.github.com/articles/syncing-a-fork/).
 8. Create a new branch at the **office-js-snippets** root folder of your local repo by running the command `git checkout -b {name_of_your_new_branch}`. (This will create and checkout the new branch. *Stay in this branch for all the remaining steps.*) Each snippet should have its own branch. Suggestion: use the name of the yaml file that you created above (without the extension) as the branch name.
-9. Decide the project folder to which your snippet should be added. All snippets must be inside the `samples` folder. The structure of the subfolders is:
+9. Decide the project folder to store your snippet. All snippets must be inside the `samples` folder. The structure of the subfolders is:
```
Comment (Rick-Kirkham) [inline README.md]:
```suggestion
9. Decide the project folder where your snippet should be stored. All snippets must be inside the `samples` folder. The structure of the subfolders is:
```

File: README.md [hunk 4]
```diff
@@ -35,12 +35,12 @@ api_set:
...
  - The base folders such as `Excel`, `Word` etc. are all the various broad-level categories.
- - Inside of each base folder, there are group folders for the group in which a snippet belongs.
- - Inside of each group folder, there are `.yaml` which represent a snippet.
+ - Group folders for the group containing a snippet are inside each base folder.
```
Comment (Rick-Kirkham) [inline README.md]:
```suggestion
 - Inside of each base folder there are group folders.
```

File: README.md [hunk 5]
```diff
@@ -35,12 +35,12 @@ api_set:
...
+ - '.yaml' representing a snippet are inside each group folder.
```
Comment (Rick-Kirkham) [inline README.md]: Inside each group folder are '.yaml' files. Each one represents a snippet.

**Comment by ElizabethSamuel-MSFT** [comment]
@tdslinden: Thanks for taking the time to contribute to this repo! I've resolved conflicts against the OfficeDev master branch.

**Pull request merged**